packages/datadog-instrumentations/src/http.js

```diff
@@ -0,0 +1,214 @@
+'use strict'
+
+const url = require('url')
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+const { storage } = require('../../datadog-core')
+const { incomingHttpRequestStart } = require('../../dd-trace/src/appsec/gateway/channels')
+const log = require('../../dd-trace/src/log')
+
+const startServerCh = channel('apm:httpServer:request:start')
+const endServerCh = channel('apm:httpServer:request:end')
+const errorServerCh = channel('apm:httpServer:request:error')
+const wrapServerCh = channel('apm:httpServer:request:wrap')
```

Comment:
This needs a better name than wrap. This is the event where we first have the request object, right? Let's name it accordingly.

Reply:
In fact this shouldn't be needed at all since it's called right after startServerCh.